system/default_ad_api/src/motion.cpp

```diff
@@ -138,6 +138,7 @@ void MotionNode::on_is_start_requested(
   const control_interface::IsStartRequested::Message::ConstSharedPtr msg)
 {
   is_start_requested_ = msg->data;
+  update_state();
```

Comment:
@isamu-takagi 
Do you see any issue with this change ?
The goal of this change is to allow the state to be updated as soon as a request to start is received. Otherwise we have to wait for the timer callback (which can take up to `100ms` since it runs at `10hz` by default).

Reply:
Thank you. Your point made me realize that other parts also need improvement. Could you please check this commit https://github.com/autowarefoundation/autoware.universe/pull/3385/commits/b8bf15664c77c68914721395dfd4c7b9569efda3? I pushed it to this branch by mistake.

Reply:
Thank you. I tested with your commit and it still works well.
Should I revert your commit ?

Reply:
> Should I revert your commit ? (in case you want to create a separate PR).

I think it's fine to merge with this PR.